File: MATLAB/ASCII_Enc_German.m
function y = ASCII_Enc_German(x)
% Very similar to preprocessing.
    n = size(x, 2);
    for i = 1 : n
        if (x(i) == 91) 
            y(i) = 196;
        elseif (x(i) == 92)
            y(i) = 214;
        elseif (x(i) == 93)
            y(i) = 220;
        elseif (x(i) == 94)
            y(i) = 223;
        else
            y(i) = x(i);
        end
    end
end

File: R/ASCII_Enc_German.r
ASCII_Enc_German <- function(ciphertext_ascii)
{
  ciphertext_ascii[ciphertext_ascii == 91] <- 196
  ciphertext_ascii[ciphertext_ascii == 92] <- 214
  ciphertext_ascii[ciphertext_ascii == 93] <- 220
  ciphertext_ascii[ciphertext_ascii == 94] <- 223
  
  return(ciphertext_ascii)
}